addons/project/static/src/project_sharing/chatter/message_model_patch.js

```diff
@@ -1,9 +1,8 @@
-import { Message } from "@mail/core/common/message";
-
 import { patch } from "@web/core/utils/patch";
+import { Message } from "@mail/core/common/message_model";
 
 patch(Message.prototype, {
-    get shouldHideFromMessageListOnDelete() {
+    shouldHideFromMessageListOnDelete(env) {
         return true;
```

Comment:
Already before this PR but bad pattern here where inclusion of the file implies the condition, rather than explicit check based on env.